.env.example

```diff
@@ -1,20 +1,20 @@
-# ------- NEEDED --------- #
-MODE="production"
+# ------- REQUIRED --------- #
```

Comment:
There were some issues with unquoted values during docker builds. Don’t remember what exactly it was though.

Reply:
Okay, I see. I'll try to modify bash script to add `"` only if it's not present. Thanks for warning!